ci/Dockerfile.photon

```diff
@@ -9,7 +9,7 @@ RUN tdnf install -q -y --enablerepo=photon-debuginfo \
                        popt-devel sed createrepo_c glib libxml2 findutils \
                        python3 python3-pip python3-setuptools python3-devel \
                        valgrind gpgme-devel glibc-debuginfo libmetalink-devel \
-                       openssl-devel which
+                       openssl-devel which git tar gzip file
```

Comment:
these dependencies are for building the rpm. You can take a call whether to install it here or have a check in the script and log message saying we need these packages for building rpm.